lib/src/clojure_lsp/kondo.clj

```diff
@@ -129,6 +137,12 @@
                 :name-end-row 0
                 :name-end-col 0))]
 
+    :var-usages
+    [(cond-> element
+       (and (full-qualified-namespace? element)
+            (not (:alias element)))
```

Comment:
I'm just a little bit concerned with performance as this function will be called for every var-usages being analyzed, maybe we could check alias first just to filter out some usages.
```suggestion
       (and (not (:alias element))
            (full-qualified-namespace? element))
```